ROADMAP.md

```diff
@@ -41,6 +42,10 @@ Objective: "Improve the Serverless Experience by reducing cold starts/stops to 1
 * Server Shutdown
     * Ensure that all model servers shutdown within 10 seconds of not receiving traffic.
 
+Objective: "Simplify User Experience."
```

Comment:
Let's make this objective more crisp. 

Simplify user experience with handling credentials for storage backends.

Reply:
done